### DISCUSSION TITLE
Is there any release schedule for pkcs11-provider ?

### DISCUSSION THREAD

**Discussion by sahilnxp** [openssl-projects/pkcs11-provider#454] [General]
Hi,

Is there any release schedule for pkcs11-provider?

Regards,
Sahil

**Comment by simo5** [comment]
pkcs11-provider releases when there is new functionality or bugfixes that are important enough.
There is no fixed time-based release schedule nor yet a feature based one.

Making a release is relatively simple for us, so if there is a bug fix or feature you can't backport and want a release for, feel free to ask in an issue and we'll see if the code is in good shape enough for an immediate release or we rather wait for something to be resolved.

Right now we are planning a release once #449 and #453 are merged.

**Reply by sahilnxp** [reply]
Thanks @simo5 for this information.
